engines/groovie/detection.cpp

```diff
@@ -132,31 +133,41 @@ static const GroovieGameDescription gameDescriptions[] = {
 				AD_LISTEND
 			},
 			Common::EN_ANY, Common::kPlatformIOS, ADGF_NO_FLAGS,
-			GUIO3(GUIO_NOMIDI, GUIO_NOASPECT, GAMEOPTION_T7G_FAST_MOVIE_SPEED)
+			GUIO4(GUIO_NOMIDI, GUIO_NOASPECT, GUIO_NOSFX, GAMEOPTION_T7G_FAST_MOVIE_SPEED)
 		},
 		kGroovieT7G, 0
 	},
 
 	// The 11th Hour DOS English
 	{
 		{
-			"11h", "",
+			"11h", "CD-ROM",
 			AD_ENTRY1s("disk.1", "5c0428cd3659fc7bbcd0aa16485ed5da", 227),
 			Common::EN_ANY, Common::kPlatformDOS, ADGF_UNSTABLE,
 			GUIO4(GUIO_MIDIADLIB, GUIO_MIDIMT32, GUIO_MIDIGM, GUIO_NOASPECT)
 		},
-		kGroovieV2, 1
+		kGroovieT11H, 1
+	},
+
+	{
+		{
+			"11h", "deu",
+			AD_ENTRY1s("disk.1", "3d5133fc457e80b80591086dce283792", 76),
```

Comment:
I would highly recommend reviewing all these detection entries and go away from "disk.1" or add another file. This filename is way too generic and that means that any game with that file would produce a false positive for "unknown 11th-hour game version".

Reply:
this will be difficult to find all these versions of all these games, should we just remove the ones we can't verify?

Reply:
groovie.cpp uses the first entry assuming that it will be disk.1
```Common::String filename = _gameDescription->desc.filesDescriptions[0].fileName;```

so I will do my detection like this
```
	// The 11th Hour Windows English (Available on Steam)
	{
		{
			"11h", "",
			AD_ENTRY2s("disk.1", NULL, -1,
				"dir.rl", "68389a565f6217bdbc6edae6938a0810", 166432),
			Common::EN_ANY, Common::kPlatformWindows, ADGF_UNSTABLE,
			GUIO5(GUIO_MIDIADLIB, GUIO_MIDIMT32, GUIO_MIDIGM, GUIO_NOASPECT, GAMEOPTION_T7G_FAST_MOVIE_SPEED)
		},
		kGroovieT11H, 1
	},
```

Reply:
No, don't remove, leave those as is. I will take a look into my archive and add the missing files if I have them. Only after this second round we will consider disabling some entry.